### ISSUE TITLE
[SITE BROKEN] Novelupdates.com - since they started using hCaptcha

### ISSUE THREAD

**Issue by frykauf** [Flameish/Novel-Grabber#277]
Novelupdates.com 

TLDR: Cloudflare captcha is a bitch
Novelupdates now requires hCaptcha and everytime you want NG to grab chapter list from Novelupdates it brokey.

**Additional information**
Well this is how it looks like when you try to get chapter list from Novelupdates link (Novel URL):

![what_is_it_novelupdates_chan](https://user-images.githubusercontent.com/8696531/131432799-d8cca3f8-c1a3-4d0d-b918-1a22f09665fd.jpg)

And when you try to refresh you get this. Even when you solve the captcha, the browser can’t get through.

![hCaptcha-get](https://user-images.githubusercontent.com/8696531/131432854-00df63a1-3875-4636-a13d-e6ba1c6ecc24.jpg)

The issue has been here for about 2+ weeks, I though it’s useless to report since the dev would know already since it’s NovelUpdates but fix isn’t here so. I’m here to report.

**Comment by Flameish** [comment]
NovelUpdates not working is known. Basically everything tagged with Cloudflare is not working because of bot detection.

**Comment by Flameish** [comment]
#220 is the NovelUpdates issue concerning this.
